pkg/jsx/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@hyperapp/jsx",
+  "description": "Hyperapp JSX runtime",
+  "version": "0.0.1",
+  "type": "module",
+  "main": "jsx-runtime.js",
+  "license": "MIT",
+  "repository": "https://github.com/jorgebucaran/hyperapp/tree/master/pkg/jsx",
```

Comment:
Should be `main/packages` after you rebase with main. ☝️

Reply:
Thanks for the heads up, rebased 👍